app/main.py

```diff
@@ -1,3 +1,60 @@
+from __future__ import annotations
+
+
 class Distance:
     # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += + other
+        return self
+
+    def __iadd__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, (int, float)):
+            self.km += other
+        else:
+            self.km += other.km
+        return self
+
+    def __mul__(self, other: int | float | Distance) -> Distance:
+        self.km = self.km * other
+        return self
+
+    def __truediv__(self, other: int | float | Distance) -> Distance:
+        self.km = round(self.km / other, 2)
+        return self
+
+    def __lt__(self, other: Distance | int | float) -> bool:
+        if isinstance(other, (Distance, int, float)):
+            return self.km < other
+        return self.km < other.km
+
+    def __gt__(self, other: int | float | Distance) -> bool:
+        if isinstance(other, (Distance, int, float)):
```

Comment:
Please fix it here and other places 
```suggestion
        if isinstance(other, (int, float)):
```